main.go

```diff
@@ -21,13 +21,14 @@ import (
 // OutputFormatVersion will allow us to filter output files downstream.
 const OutputFormatVersion = '1'
 
-// CPUTimesStat is a more limited version of cpu.TimesStat to save storage space
-type CPUTimesStat struct {
-	User   float64 `json:"user"`
-	System float64 `json:"system"`
-	Idle   float64 `json:"idle"`
-	Iowait float64 `json:"iowait"`
-	Steal  float64 `json:"steal"`
+// ProcCPUStat is a more limited version of cpu.TimesStat to save storage space
+type ProcCPUStat struct {
+	User    float64 `json:"user"`
+	System  float64 `json:"system"`
+	Idle    float64 `json:"idle"`
+	Iowait  float64 `json:"iowait"`
+	Steal   float64 `json:"steal"`
+	Percent float64 `json:"percent"`
```

Comment:
I debated whether to include this in a separate struct but made more sense to make the CPU collection as a `Proc`related item, as are the other things we're collecting. I'm open to whether to move it separately should you want.